governance/votes/Executive vote - July 11, 2023.md

```diff
@@ -0,0 +1,122 @@
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Smart Burn Engine Deployment
+
+As per [this poll](https://vote.makerdao.com/polling/QmQmxEZp), the Smart Burn Engine will be implemented and the old flapper Depreciated. The following actions will occur
+
+- Depreciate MCD_FLAP
+- Deploy [FlapperUniV2](https://github.com/makerdao/dss-flappers/blob/univ2/README.md)
+- Reduce vow.hump by 200 million DAI from 250 million DAI to 50 million DAI
+- Reduce vow.bump by 25,000 DAI from 30,000 DAI to 5,000 DAI
+- Set hop to 1,577 seconds
+- Set want to 0.98
+- Set receiver to MCD_PAUSE_PROXY
+- Set pip to TBD
+
+### Stability Scope Parameter Changes
+
+As per the Stability Scope Facilitator, SES, [the following changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238) will be implemented. 
+
+#### [Dai Savings Rate(DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Decrease
+
+- Reduce DSR by 0.30% from 3.49% to 3.19%
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Decreases
+
+- Reduce the ETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the ETH-B Stability Fee (SF) by 0.30% from 4.24% to 3.94%
+- Reduce the ETH-C Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the WSTETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WSTETH-B Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the RETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WBTC-A Stability Fee (SF) by 0.11% from 5.80% to 5.69%
+- Reduce the WBTC-B Stability Fee (SF) by 0.11% from 6.30% to 6.19%
+- Reduce the WBTC-C Stability Fee (SF) by 0.11% from 5.55% to 5.44%
+
+#### [Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) Decreases
+
+- Reduce WSTETH-A Liquidation Ratio by 10% from 160% to 150%
+- Reduce WSTETH-B Liquidation Ratio by 10% from 185% to 175%
+- Reduce RETH-A Liquidation Ratio by 20% from 170% to 150%
+
+#### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+-  Set CRVV1ETHSTETH-A [Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) to 0
+-  Remove CRVV1ETHSTETH-A from autoline
```

Comment:
Don't refer to it as autoline in these docs, use the human readable name, and link to the docs.